certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -2455,6 +2578,82 @@
         "description": "Provides the dynamic schema for the Create Barcode action."
       }
     },
+    "/api/v1/DynamicSchema/GetDynamicSchemaConvertWord": {
+      "get": {
+        "tags": [
+          "DynamicSchema"
+        ],
+        "operationId": "GetDynamicSchemaConvertWord",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "outputFormat",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "Output Format",
+            "description": "The output format to obtain the dynamic schema for."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object"
+            }
+          },
+          "400": {
+            "description": "Bad request"
+          }
+        },
+        "x-ms-visibility": "internal",
+        "summary": "GetDynamicSchemaConvertWord",
```

Comment:
@Encodian Summary value is provided as an operation Id. As it is visible to public, please provide a valid sentence which suits the operation functionality.